com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -432,163 +494,141 @@ void Send(NetworkTransformState stateToSend)
             // making it immobile.
             if (isDirty)
             {
-                Send(m_LocalAuthoritativeNetworkState);
+                // Commit the state
+                ReplicatedNetworkState.Value = m_LocalAuthoritativeNetworkState;
                 m_HasSentLastValue = false;
                 m_LastSentTick = m_CachedNetworkManager.LocalTime.Tick;
                 m_LastSentState = m_LocalAuthoritativeNetworkState;
             }
             else if (!m_HasSentLastValue && m_CachedNetworkManager.LocalTime.Tick >= m_LastSentTick + 1) // check for state.IsDirty since update can happen more than once per tick. No need for client, RPCs will just queue up
             {
+                m_LastSentState.IsTeleportingNextFrame = false; // This is required here
```

Comment:
"This is required here" - can you elaborate in the comment, so when we come back to this later, we not only know that it's required, but why?

Reply:
Heh... yeah... I was wondering if I needed to elaborate on that or not.
Basically, that portion of the code is just resetting the IsTeleportingNextFrame because of the m_LastSentState as it is used (based on the legacy notes above it) to help clients extrapolate to the last position (i.e. making them motionless) if there are no more deltas to be sent.
This is called out in the document and I think this "else if" segment of code can be removed completely (it requires us to migrate this functionality over to the non-authoritative side).

Reply:
I meant, can you add this to the comment in the code, so that if someone comes here to refactor this later (and doesn't know about the document) they can understand the reasons that has to be right there specifically?